### ISSUE TITLE
List of projects using ROBOT

### ISSUE THREAD

**Issue by jamesaoverton** [ontodev/robot#359]
A list of projects using ROBOT would be helpful. I'm specifically thinking of ontology projects that use ROBOT, but there are other connections like ODK.

Here are some off the top of my head, in no particular order:

- GO
- Uberon
- OBI
- DO
- ECO
- ENVO
- MRO

Please comment with other you know about!

**Comment by balhoff** [comment]
NCIT OBO Edition

**Comment by rctauber** [comment]
Should we maintain a doc page with the list of projects?

**Comment by jamesaoverton** [comment]
I don't think that I want to be bothered maintaining a list in our docs. This information is mostly useful for grant writing and came in handy for me this week.

The number I care about most is the percentage of active OBO projects using ROBOT.

**Comment by cmungall** [comment]
In theory we should be able to do this with a github query. I think most
people commit their workflow in some form or another - script, Makefile, or
at least a markdown doc
